src/components/Wizard/CreateVmWizard/providers/v2vvmwareController.js

```diff
@@ -11,161 +14,33 @@ export const startV2VVMWareController = async ({ k8sCreate, k8sGet, namespace })
   try {
     await k8sGet(DeploymentModel, V2VVMWARE_DEPLOYMENT_NAME, namespace);
   } catch {
-    // Deployment does not exist or the does not have permissions to see Deployments in this namespace
+    // Deployment does not exist or does not have permissions to see Deployments in this namespace
     // TODO: notify the user in other way not just the console.log
 
-    // eslint-disable-next-line no-console
-    console.info('V2V VMWare controller deployment not found, so creating one ...');
-    const params = { k8sCreate, namespace };
+    info('V2V VMWare controller deployment not found, so creating one ...');
+    const params = { k8sCreate, namespace, name: V2VVMWARE_DEPLOYMENT_NAME };
 
-    /* eslint-disable promise/catch-or-return */
-    createServiceAccount(params).then(undefined, rejectLogger('Service Account'));
-    createRole(params).then(undefined, rejectLogger('Role'));
-    createRoleBinding(params).then(undefined, rejectLogger('Role Binding'));
-    createOperator(params).then(undefined, rejectLogger('V2V VMWare controller'));
-    /* eslint-enable promise/catch-or-return */
+    createServiceAccount(params).catch(rejectLogger('Service Account'));
+    createRole(params).catch(rejectLogger('Role'));
+    createRoleBinding(params).catch(rejectLogger('Role Binding'));
+    createOperator(params).catch(rejectLogger('V2V VMWare controller'));
   }
 };
 
-const createServiceAccount = ({ k8sCreate, namespace }) =>
```

Comment:
I am not big fan of melting this code into the rest of our code-base as it sends incorrect message for other contributors.

My original intention was to keep it isolated as it should not be part of UI at all.
We just don't have better place to put it now.
As soon as it is found, we should remove it.

Reply:
Yup, it is not good but we need a place for that and have some separation of code for now. We already have v2v in requests.

Reply:
I am afraid that we will have hard time to get rid of it in the future ...
I still prefer collocation this "unrelated" stuff somewhere.

Reply:
It is already getting used in two places, so I would rather have a proper place for that in the project structure (reusability). I know it is not good to overuse this stuff in the UI, but we can enforce that in merging process.